File: server/hudie/hudie/message/Protocols/wu_liao/bored_record_item_add_req.cs
using messages;

namespace messages.Protocols
{
  public class bored_record_item_add_req : MsgBase
  {
      public bored_record_item_add_req()
      {
          CodeId = MsgCodeId.bored_record_item_add_req;
      }
      public string head_id;    //
      public string url;    //
      public int time;    //

	}
}

File: client/butterfly/app/src/main/java/pf/com/butterfly/message/Protocols/bored_record_item_add_req.java
package pf.com.butterfly.message.Protocols;
import android.util.Log;
import org.json.JSONArray;
import org.json.JSONObject;
import pf.com.butterfly.message.MsgBase;
import pf.com.butterfly.message.MsgCodeId;
  public class bored_record_item_add_req extends MsgBase
  {
      public bored_record_item_add_req()
      {
          CodeId = MsgCodeId.bored_record_item_add_req;
      }
      public String head_id;    //
      public String url;    //
      public int time;    //

}
